airbyte-cdk/bulk/toolkits/load-db/src/main/kotlin/io/airbyte/cdk/load/orchestration/legacy_typing_deduping/TypingDedupingDestinationInitialStatus.kt

```diff
@@ -0,0 +1,54 @@
+/*
+ * Copyright (c) 2025 Airbyte, Inc., all rights reserved.
+ */
+
+package io.airbyte.cdk.load.orchestration.legacy_typing_deduping
+
+import io.airbyte.cdk.load.orchestration.DestinationInitialStatus
+import java.time.Instant
+
```

Comment:
this is mostly identical to the old DestinationInitialStatus class, but cleaned up for kotlin style.